aspnetcore-webapi/AddEndpointsApiExplorerMethod/AddEndpointsApiExplorerMethod/Program.cs

```diff
@@ -0,0 +1,30 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+app.MapGet("/", () => "A test endpoint");
```

Comment:
It's a good practice to declare API after all the middlewares are declared (UseXXX) , I suggest  we move this after app.UseAuthorization();

Reply:
Sure, this is done.